src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -6,71 +6,87 @@
 
 import java.awt.image.BufferedImage;
 import java.util.function.Function;
+import java.util.Arrays;
 import java.util.EnumSet;
 import java.util.Set;
 
 public enum ToaMistake {
     // Deaths
-    DEATH("Death", (config) -> "", "death.png"), // Chat message handled in deaths below
-    DEATH_HET("Path of Het Death", (config) -> config.deathMessage(), "death-akkha.png"),
-    DEATH_CRONDIS("Path of Crondis Death", (config) -> config.deathMessage(), "death-zebak.png"),
-    DEATH_SCABARAS("Path of Scabaras Death", (config) -> config.deathMessage(), "death-kephri.png"),
-    DEATH_APMEKEN("Path of Apmeken Death", (config) -> config.deathMessage(), "death-baba.png"),
-    DEATH_WARDENS("Wardens Death", (config) -> config.deathMessage(), "death-wardens.png"),
+    DEATH("Death", (config) -> "", "I'm planking!", "death.png"), // Chat message handled in deaths below
+    DEATH_HET("Path of Het Death", (config) -> config.deathMessage(), "I'm planking!", "death-akkha.png"),
```

Comment:
nit: We could replace the lambda with the method reference (here and everywhere else): 
```
ToaMistakeTrackerConfig::deathMessage
```
Doesn't matter much either way, but it's a bit easier to read and know exactly which method and from which class this is coming from.